shenyu-client/shenyu-client-sofa/src/main/java/org/apache/shenyu/client/sofa/SofaServiceBeanPostProcessor.java

```diff
@@ -75,7 +76,8 @@ public SofaServiceBeanPostProcessor(final ShenyuRegisterCenterConfig config, fin
         this.appName = appName;
         this.host = props.getProperty("host");
         this.port = props.getProperty("port");
-        executorService = new ThreadPoolExecutor(1, 1, 0L, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<>());
+        executorService = new ThreadPoolExecutor(1, 1, 0L, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<>(), new ThreadFactoryBuilder()
+                .setNameFormat("sofa-client-thread-pool-%d").build());
```

Comment:
if u want to add poll name, I think u must add prefix like 'shenyu-' first to avoid misunderstanding .

Reply:
ok, let me change it.